### ISSUE TITLE
Devices Showing as 'No responding' on other home users 

### ISSUE THREAD

**Issue by shanebowler** [mbudnek/google-home-hubitat-community#46]
**Describe the bug**
I have had my account linked to the google community app for a while and I have had 2 other members invited and using my home. Recently when adding a couple of new devices, they were working fully on my google home as expected (the one that's linked to the test app I created) however when looking at the app on the other home users, the bulb displays, but it shows as 'not responding' I have used the 'sync my devices' I have deleted the link to the app, I have even gone to the extreme of deleting the my home and re-building the test app on a separate account.

The simple way to resolve this, is to get the user to leave the home then I re-invite them, this seems to kick start the device into being controllable via other google home users. However when I again add a new device the new device displays the same traits.

**Comment by shanebowler** [comment]
Weird observations are: 
1 If I change the device type in the Hubitat App, it no longer becomes 'not responding' on the other apps. I can then change it back to 'light' and it works.
2 - If I rename the light in the other user google home device, it then becomes responsive. I can then rename it back and it works

**Comment by mbudnek** [comment]
I really don't have any idea why that would be happening.  If you turn on debug logging in the Hubitat app, do you see anything in the logs when someone else you've shared the non-responsive devices with tries to run a command on them (i.e. using voice control on their phone)?

**Comment by shanebowler** [comment]
When refreshing google home by dragging down on the phone it dials back to the app, I can see it in the logs, but clicking the on and off button doesn't do anything or dial back to the app, nor does it when I ask google to to turn on the light...It's very odd, especially as it works when re-inviting users back into the home

**Comment by shanebowler** [comment]
Hey,

Just to update you as I've resolved my issue. So I have always had my smart home on an IoT vlan and even the Google Community App. For whatever reason (even though I have made no changes) that's what was causing the issue. In hindsight I could have restarted my network, but as a test I moved it to the Home Vlan and it has resolved. I wonder if an update to Hubitat, or something has blocked a port or blocked the updating the status.
